src/multibody/geometry.hxx

```diff
@@ -42,10 +43,9 @@ namespace pinocchio
   GeomIndex GeometryModel::addGeometryObject(const GeometryObject & object,
                                              const ModelTpl<S2,O2,JointCollectionTpl> & model)
   {
-    assert( //TODO: reenable when relevant (object.parentFrame == -1) ||
-           (model.frames[object.parentFrame].type == pinocchio::BODY)  );
```

Comment:
I don't remember why the added this assert. I think the rationale was to check data coherence. If it is not a body, the caller likely made a mistake.

Reply:
Yes,  but as `parentFrame` is optional, I think this check is not relevant and may fail.